airbyte-integrations/connectors/source-dentclinic/source_dentclinic/source.py

```diff
@@ -0,0 +1,144 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+
+from abc import ABC
+from typing import Any, Iterable, List, Mapping, Optional, Tuple, Iterator, MutableMapping
+
+import requests
+import pendulum
+import xmltodict
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import Stream
+from airbyte_cdk.sources.streams.http import HttpStream
+
+from .streams import DentclinicBookingStream, DentclinicBookingFrStream, DentclinicClinicIdsStream, DentclinicStaticStream
+
+
+class BookingsFr(DentclinicBookingStream):
```

Comment:
Think we need to use DentclinicBookingFrStream instead with fr_days_forward and fr_days_back configd

Reply:
Added `days_back` parameter to config